bitsofgood/pencils-for-success
# Jason/165/update donor dashboard

### Description
- Add image to Donor Dashboard
- Style side panel and main panel
- Style dropdown button
- Style toggle button

**Related Issues/PRs:** 
- Closes #165 

### Test Plan
- Toggle map and info view
- Select a different chapter
- Confirm frontend updates
